library.properties

```diff
@@ -1,5 +1,5 @@
 name=Adafruit MQTT Library
-version=2.4.3
+version=2.4.4
```

Comment:
is this an appropriate "patch" level update? it does change how the code acts, if people were relying on the broken behavior and have worked around it themselves, this will break their clients. too pedantic of me?

Reply:
@lorennorman  not too pedantic, maybe 2.5.x is better.